### PR TITLE
HCG color model support 

### PR DESCRIPTION
**Pull request by acterhd**
Hello. Please, add this color model to "color-space.js".

### CODE REVIEW

File: hcg.js [hunk 1]
```diff
@@ -0,0 +1,124 @@
+/**
+ * @module color-space/hsv
+ */
+
+var rgb = require('./rgb');
+var hsl = require('./hsl');
+var hsv = require('./hsv');
+
+module.exports = {
+	name: 'HCG',
```
Comment (dfcreative) [inline hcg.js]: Pls lowcap this and add 'HCG' to alias

**Comment by dfcreative** [comment]
@acterhd please also add a line with link to the readme

**Comment by acterhd** [comment]
Fixed.

2016-02-01 7:39 GMT+08:00 Δॐma Yvanow <notifications@github.com>:

**Comment by acterhd** [comment]
Sorry, I'm slept.

2016-02-01 14:59 GMT+08:00 Alexey Solovey <acterhd@gmail.com>:

**Comment by dfcreative** [comment]
@acterhd thanks!

**Pull request merged**